### ISSUE TITLE
Less verbose logs on func start

### ISSUE THREAD

**Issue by pragnagopa** [Azure/azure-functions-core-tools#2141]
Part of #1131 

This issue tracks following:

(copied from https://github.com/Azure/azure-functions-core-tools/issues/1131#issuecomment-643655379)

## func start

* Change default to a less verbose output, hiding almost everything coming from host or Core Tools that is `Information` or `Debug`
* Add a `--verbose` flag that outputs the current (original) log levels
* Remove timestamps from output when not verbose
* Display a summary of *all* functions, not just HTTP
* Output a message to indicate host is ready (required by VS Code to attach debugger)
* **Update:** Remove logo 😱

```
$ func start

Azure Functions Core Tools (3.0.2534 Commit hash: bc1e9efa8fa78dd1a138dd1ac1ebef97aac8d78e)
Function Runtime Version: 3.0.13353.0

Functions:

        HttpTrigger: [GET,POST] http://localhost:7071/api/HttpTrigger

        AnotherHttpTrigger: [GET,POST] http://localhost:7071/api/AnotherHttpTrigger

        MyTimerTrigger: TimerTrigger (maybe list out the next 5 invocation times?)

        SomeOtherTrigger: QueueTrigger

Host started. For detailed output, run func with the --verbose flag.
```

Refer to sample here: https://github.com/Azure/azure-functions-core-tools/issues/1131#issuecomment-653171101

Proof of concept can be found here:
https://github.com/Azure/azure-functions-core-tools/compare/7dd097e...anthonychu:less-verbose

**Comment by pragnagopa** [comment]
Addressed in referenced PR #2139

**Comment by pragnagopa** [comment]
@FinVamp1 - FYI this change will be going out with the next release.
